syncapi/storage/interface.go

```diff
@@ -30,6 +28,26 @@ import (
 
 type Database interface {
 	internal.PartitionStorer
+
+	MaxStreamTokenForPDUs(ctx context.Context) (types.StreamPosition, error)
```

Comment:
Token or Position? We should be consistent in our terminology and I think you mean Position.

Reply:
Fixed - it's Position.